contrib/build-linux/appimage/Dockerfile_ub2204

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:22.04@sha256:bbf3d1baa208b7649d1d0264ef7d522e1dc0deeeaaf6085bf8e4618867f03494
```

Comment:
Yeah as @SomberNight said basically the libc version the binaries in the appimage expect must be older than or equal to the system the appimage is expected to execute on, otherwise nothing works.

Reply:
I'll see which older versions will work.  1804 did not, because of a conflict in typing-extensions.

Reply:
Looks like it will work OK with 20.04.  So I will work on a new pull request.

Reply:
I think ubuntu 20.04 is old enough to upgrade to.
Still, I am curious, what is it that is difficult to build specifically?

Reply:
The new Trezor model 3 requires a newer trezor library, a pip install.  This version of the library changed the interface to the pip module typing-extensions (aka typing_extensions) which changed the Python type of one interface parameter.  After many tries and experimenting with various versions of calling out typing-extensions it was not possible to find one that would work with the new trezor library. The best that I could do was to have a build that completed and it would run OK until it encountered a wallet that used a Trezor (or an attempt to create one). At that point one would see error messages indicating the version conflict.  As I recall, the error message was paradoxical in its description of the version number conflict.  I suspect that the root of the problem is that the newer typing-extension releases were not back ported to Ubuntu 18.04.

Thank you for pointing out the problem and educating me.  After spinning up an Ubuntu 20.04 VM yesterday, I discovered that the result of this pull request was an appimage that would not run on this system.  I have a new pull request in the works that needs some more testing with different OS versions and all my Trezors.

Reply:
I am going to close this pull request.  It will be replaced shortly with a new pull request based of of an Ubuntu 20.04 Docker app image build.